shared/mail/mail_test.go

```diff
@@ -368,8 +368,8 @@ func TestSendMail(t *testing.T) {
 			"",
```

Comment:
We also need to change the name of the test here. Something like "adds message-id header always"